io/plugins/eu.esdihumboldt.hale.io.json/src/eu/esdihumboldt/hale/io/json/JsonInstanceReader.java

```diff
@@ -58,17 +86,61 @@ protected IOReport execute(ProgressIndicator progress, IOReporter reporter)
 		progress.begin("Creating " + getDefaultTypeName() + " parser", ProgressIndicator.UNKNOWN);
 
 		try {
-			boolean expectGeoJson = true; // currently defaults to true, no
-											// major difference in functionality
+			boolean expectGeoJson = false;
```

Comment:
Not sure what you want to achieve with all the changes here. If it is about supporting using the information in `@type`, then that would be better suited to be done in `JsonToInstance` for each individual instance.